### DISCUSSION TITLE
Capstone Flask: keluar error ketika memilih terminal di studio code

### DISCUSSION THREAD

**Discussion by annerina84** [teamalgoritma/community#503] [Q&A] [answered]
Mhn bantuannya, ketika akan me-run app.py, ketika memilih "new terminal" malah error,
![image](https://user-images.githubusercontent.com/120439978/211179631-50612fa5-b464-4eda-b897-3a5e789ce624.png)

**Comment by LyciaPricia** [comment, marked as answer]
coba pilihnya jangan powersheel tapi command prompt aja. bisa dipilih dengan menggunakan tanda + diatasnya.

**Reply by annerina84** [reply]
Wah mba Lycia, terimakasih banyak....dr tadi dah gonta ganti laptop, dikira ada yang salah. thanks ya Mba Lycia... solved

**Reply by LyciaPricia** [reply]
no problem..happy to help
